webapp/views/App/views/Data/Charts/components/Chart/Chart.js

```diff
@@ -4,30 +4,39 @@ import Split from 'react-split'
 import Data from './components/Data'
 import ScatterPlot from './components/ChartTypes/ScatterPlot/'
 import BarChart from './components/ChartTypes/BarChart/'
+import PieChart from './components/ChartTypes/PieChart/'
 import './Chart.scss'
 
-const Chart = ({ data, specs, draft, renderChart, fullScreen }) => {
+const Chart = ({ data, specs, fullScreen }) => {
   const chartType = specs?.chartType
   const hasData = Boolean(data)
   const hasSvg = Boolean(data?.svg)
 
-  const ChartComponent = chartType === 'scatterPlot' ? ScatterPlot : BarChart
+  let ChartComponent
+  switch (chartType) {
+    case 'scatterPlot':
+      ChartComponent = ScatterPlot
+      break
+    case 'barChart':
+      ChartComponent = BarChart
+      break
+    case 'pieChart':
+      ChartComponent = PieChart
+      break
+    default:
+      ChartComponent = null
+  }
```

Comment:
this could be simplified with an object/dictionary like:
chartComponentByType: {
    scatterPlot: ScatterPlot,
    ...
}

and it would be nice to have the chart types defined as costants